modules/template/controllers/ContainerContentController.php

```diff
@@ -261,4 +261,45 @@ public function actionMoveItem($elementContentId, $itemId, $step)
         ]);
     }
 
+    public function actionExportInstance()
+    {
+        $model = TemplateInstance::findOne(['id' => Yii::$app->request->get('id')]);
+
+        if ($model === null) {
+            throw new NotFoundHttpException(Yii::t('CustomPagesModule.template', 'Template instance is not found!'));
+        }
+
+        return ExportInstanceService::instance($model)->export()->send();
+    }
+
+    public function actionImportInstance()
+    {
+        $instance = TemplateInstance::findOne(['id' => Yii::$app->request->get('id')]);
```

Comment:
Access Check?

Reply:
@yurabakhtin Same as above. I think it is possible to import all template instances here, even to other spaces.

Reply:
Same answer as [here](https://github.com/humhub/custom-pages/pull/407#discussion_r2113800394)